src/main/java/edu/iu/terracotta/service/app/impl/ExportServiceImpl.java

```diff
@@ -148,7 +148,7 @@ public Map<String, List<String[]>> getCsvFiles(Long experimentId, SecuredInfo se
                         if (!StringUtils.isAllBlank(egc.getCondition().getName())) {
                             conditionName = egc.getCondition().getName();
                         }
-                        List<Assignment> assignments = allRepositories.assignmentRepository.findByExposure_ExposureId(egc.getExposure().getExposureId());
+                        List<Assignment> assignments = allRepositories.assignmentRepository.findByExposure_ExposureIdAndSoftDeleted(egc.getExposure().getExposureId(), false);
```

Comment:
I'm wondering if we should include soft deleted assignments that have submissions (which is all soft deleted assignments since they get hard deleted if no submissions). I'll ask on Jira.